online-shop/src/app/components/presentational/products-list-view/products-list-view.component.scss

```diff
@@ -22,8 +22,8 @@
     text-align: left;
   }
 }
-/* Style for to Cart and button */
-.to-cart-button {
+
+.to-cart-button,.add-button{
```

Comment:
From what I know you do not need to separate the scss classes by comma `,`. You can simply enumerate them with spaces between their names

Reply:
Removed the comma.